### PR TITLE
Implements Payment by link

### LINKED ISSUE #331: Payment by link (old Request Payment)
#### Expected behavior and actual behavior.
We have a "poor" experience to create a payment manually from plugin, something was created on this issue: https://github.com/ebanx/woocommerce-gateway-ebanx/issues/85

We need to launch a new feature with a new flow to the merchant create a payment directly from plugin.

### PR DESCRIPTION
**Pull request by gpressutto5**
Closes #331

### CODE REVIEW

File: woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php [hunk 1]
```diff
@@ -724,6 +729,155 @@ public function ebanx_sidebar_shortcut()
...
+			);
+
+			$config = array(
+				'integrationKey' => $this->private_key,
+				'testMode' => false
+			);
+
+			\Ebanx\Config::set($config);
+
+    		$request = \Ebanx\EBANX::doRequest($data);
```
Comment (gpressutto5) [inline woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php]: @cezarlz What is it talking about?

**Comment by cezarlz** [comment]
@gpressutto5 what is the progress of this task? Can I help you?

**Comment by gpressutto5** [comment]
Should we ignore @codacy-bot on this pull request?
@SparK-Cruz @cezarlz @guilhermepiovesan

**Comment by SparK-Cruz** [comment]
I guess... the code IS clean, you can read the validations and everything is clear on what things do.
I'd say yes, but wait for others.

**Comment by SparK-Cruz** [comment]
Codacy is positive now...

**Pull request merged**